scripts/github-actions-release.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e # Turn on exit on error
+
+# Set default AWS region and tag
+
+#
+######## update ECS service with new image ################
+#
+aws ecs update-service --cluster $DEPLOYMENT_ENV-microservice-cluster  --service ab2d-contracts-service --force-new-deployment
```

Comment:
I would rather see this in the workflow directly rather than wrapped in a script. It's an unnecessary abstraction layer and makes troubleshooting from the logs more difficult.